### DISCUSSION TITLE
feature request: passing access logs to crowdsec

### DISCUSSION THREAD

**Discussion by darkclip** [ZoeyVid/NPMplus#631] [Q&A] [answered]
I'm coming from old npm, and I have to say great job fixing all those bugs and adding new features.
It has crowdsec support, but the access logs are disabled by default.
Seems the only way to output access logs is to go through each proxied site individually, and also there's no easy way to get stream logs.
It would be nice to have option to easily open access logs.

possible solution:
set for nginx.conf:
```
error_log /var/log/nginx/error.log;
access_log /var/log/nginx/access.log;
``` 

in startup script check for env variables
set
```
ln -sf /dev/null /var/log/nginx/error.log
ln -sf /dev/null /var/log/nginx/access.log
```
or
```
ln -sf /dev/stdout /var/log/nginx/error.log
ln -sf /dev/stdout /var/log/nginx/access.log
```
or leave the log files as is.

**Comment by Zoey2936** [comment]
see compose.yaml

**Comment by darkclip** [comment]
@Zoey2936  I think you mean the `LOGROTATE` option, yes, i see that now.
but it still doesn't output stream logs, and you wouldn't be able to use crowdsec's docker acquis option, have to use file volumes.

**Reply by Zoey2936** [reply]
I will add an aditional option to enable stream logs

**Reply by Zoey2936** [reply]
included in next release

**Comment by darkclip** [comment]
after some simple test, it seems crowdsec wouldn't able to recognize the modified log_format.
cscli shows those logs as unparsed.
So, the problem remains.

**Reply by Zoey2936** [reply]
I will try to add NPMplus as collection to parse the access log

**Reply by Zoey2936** [reply]
see https://github.com/crowdsecurity/hub/pull/967

**Reply by Zoey2936** [reply, marked as answer]
see https://app.crowdsec.net/hub/author/ZoeyVid/collections/npmplus
